br/pkg/restore/client.go

```diff
@@ -817,13 +818,8 @@ func (rc *Client) GoValidateChecksum(
 				if !ok {
 					return
 				}
+
 				workers.ApplyOnErrorGroup(eg, func() error {
-					start := time.Now()
-					defer func() {
-						elapsed := time.Since(start)
-						summary.CollectDuration("restore checksum", elapsed)
-						summary.CollectSuccessUnit("table checksum", 1, elapsed)
```

Comment:
removing this also removes the "table checksum success count" though 🤔

Reply:
it looks 
						summary.CollectSuccessUnit("table checksum", 1, elapsed)
removing the "table checksum success count" also.

no sure what else I can do?

Reply:
we keep total checksum success unit.